Lib/random.py

```diff
@@ -248,7 +248,7 @@ def randint(self, a, b):
         return self.randrange(a, b+1)
 
     def _randbelow_with_getrandbits(self, n):
-        "Return a random int in the range [0,n).  Raises ValueError if n==0."
+        "Return a random int in the range [0, n].  Raises ValueError if n==0."
```

Comment:
I don't think this is a typo. The docstring is using the mathematical notation to indicate the fact that resulting random integer can be of the boundary 0 <= result < n. This change would cause the docstring to mean 0 <= result <= n and the function never seems to return a value where result == n.


```
>>> import random
>>> list(filter(lambda x: x == 2, [random.Random()._randbelow_with_getrandbits(2) for _ in range(10000)]))
[]
```